blocks/test/fixtures/core__chat.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:core/chat {"compact":false} -->
+Pascal: Alexa, open the window
+Alexa: I'm not quite sure how to help you with
+Pascal: Siri, tell me a joke
+Siri: Pascal, get Siri-ous. Ha ha!
+Pascal: OK, Google. Now it's your turn!
+Google Home: Alexa, tell Siri to tell Pascal a better joke.
+Pascal: Ugh!
+Okay, that's enough. No internet for you anymore!
+Alexa: 💔
+Google Home: 😔
+Siri: 📴
```

Comment:
Ok Maybe instead of storing the value in its currenf format, we should extract a more "normalized" value, something like:

```js
content: query( 'p`, {
  authorId: prop( 'className' ),
  author: html( '.chat-author' ),
  message: hmlt( '.chat-message' ),
} )
```

which I think should return something like:

```js
const content = [
  { authorId, author, message }
];
```

And when typing in the text area, parsing the textarea value to the format above before calling the `setAttributes`